### DISCUSSION TITLE
Can we use Gauge Automation without Gradel Command ?

### DISCUSSION THREAD

**Discussion by suman-gs** [getgauge/gauge#2132] [General]
Hi  Team,

Currently i am using gradel dependencies in my automation project, is it posible to run gauge without "**gradel gauge PspecsDir=specs/test.spec**" command

Please let me know ,your comments on this 
getting below error when i am using  "**gauge run specs/test.spec**" command

![image](https://user-images.githubusercontent.com/92427469/137116358-3aa6caa7-653e-45db-91de-ba944b2b0ed5.png)

With Thanks
GS

**Comment by zabil** [comment]
If your project has dependencies they need to be loaded using Gradle before running a specification. You can download all the dependencies and add it to Gauge's lib folder but that's not recommended.

Your issue might be related to something else and not Gradle. Perhaps you can share more info on why you see this issue, or create a sample project  to replicate it.
